draft-ietf-quic-ack-frequency.md

```diff
@@ -253,7 +253,7 @@ Ack Delay field.
 
 # Multiple ACK_FREQUENCY Frames {#multiple-frames}
 
-An endpoint can send multiple ACK_FREQUENCY frames, and each one of them can
+An endpoint can send multiple ACK_FREQUENCY frames in the same connection, and each one of them can
```

Comment:
```suggestion
An endpoint can send multiple ACK_FREQUENCY frames on the same connection, and each one of them can
```

Not sure if this is necessary, where else would you send them? Doesn't hurt either though.